src/weas_widget/plugins/instanced_mesh_pritimive.py

```diff
@@ -0,0 +1,15 @@
+from ..base_class import WidgetWrapper
+
+
+class InstancedMeshPrimitive(WidgetWrapper):
```

Comment:
I have to admit I don't quite understand what `Instanced` means here...
why is this wrapper class "instanced" while others are not?

Reply:
[InstancedMesh](https://threejs.org/docs/#api/en/objects/InstancedMesh) is a special mesh in the three js library. It's useful when users want to create a large number of objects with the same geometry and material, but different positions/scales/colors etc. It does not have special meaning on the wrapper class.

Reply:
In that case `InstancedMeshPrimitive` is fine as a developer-facing name but I would not expose this in the API that the widget user sees (they won't understand what it means)

Reply:
Since some advanced users may want to draw some custom objects, I leave it here for the moment. Maybe move it to other namespaces in the future.